tests/host/common/mock.h

```diff
@@ -55,19 +55,23 @@
 #define D7 7
 #define D8 8
 
+#include <stdlib.h>
 #include <stddef.h>
 
+#ifndef HAVE_NONISO
+#include <stdlib_noniso.h>
```

Comment:
promised by TODO

overall libc stdlib comparison is tricky, don't see any feature flags with glibc :shrug:
matching attributes with latest version breaks old one and vice versa, so these have to be hidden unless asked for